pkgs/tools/networking/discord-sh/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib, stdenvNoCC, fetchFromGitHub, makeWrapper, curl, jq, coreutils }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "discord-sh";
+  version = "unstable-2022-05-19";
+
+  src = fetchFromGitHub {
+    owner = "ChaoticWeg";
+    repo = "discord.sh";
+    rev = "6aaea548f88eb48b7adeef824fbddac1c4749447";
+    sha256 = "sha256-RoPhn/Ot4ID1nEbZEz1bd2iq8g7mU2e7kwNRvZOD/pc=";
+  };
+
+  # The Makefile supplied by upstream wants to contact the internet, but
+  # mkDerivation wants to use it in the buildPhase by default. so we disable the
+  # buildPhase to ignore the makeFile.
```

Comment:
```suggestion
  # makefile tries to download binaries from the internet for linting
```

Reply:
I have shortened it to something that still specifies that we are disabling the buildPhase, whilst including your comment.